app/Services/MailService.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Services;
+use App\Mail\InformUserProfile;
+use Illuminate\Support\Facades\Mail;
+
+
+class MailService
+{
+    public function sendUserProfile($user)
+    {
+        Mail::to($user->all()[array_key_first($user->all())]['email'])->send(new InformUserProfile($user));
```

Comment:
Mail::to($user->all()[array_key_first($user->all())]['email'])->send(new InformUserProfile($user));

=>

Mail::to($user->email)->send(new InformUserProfile($user));